packages/ember-test-app/app/templates/cards.hbs

```diff
@@ -1,15 +1,126 @@
-<div class="container">
-  <p class="py-3">Row on desktop, stacked on mobile</p>
-  {{!-- template-lint-disable no-inline-styles --}}
-  <Mktg::CardContainer class="bg-info rounded" style="--bs-gap: .5rem;" as |Container|>
-    <Container.Card class="g-col-12 g-col-md-4" />
-    <Container.Card class="g-col-12 g-col-md-4" />
-    <Container.Card class="g-col-12 g-col-md-4" />
-  </Mktg::CardContainer>
-  <p class="py-3">stacked on mobile and desktop</p>
-  <Mktg::CardContainer class="bg-secondary" as |Container|>
-    <Container.Card class="g-col-12" />
-    <Container.Card class="g-col-12" />
-    <Container.Card class="g-col-12" />
+<div data-theme="marketing" class="container">
+  <p class="py-3 fw-bold">Row on desktop, stacked on mobile</p>
+  {{! template-lint-disable no-inline-styles }}
+  <Mktg::CardContainer style="--bs-gap: .5rem;" as |Container|>
+    <Container.Card @onClick={{this.clickHandler}} class="g-col-12 g-col-md-4">
+      <:card-header>
+        <p>Here is a title</p>
+      </:card-header>
+      <:card-body>
+        <p>Here is some body content</p>
+      </:card-body>
+    </Container.Card>
+    <Container.Card @onClick={{this.clickHandler}} class="g-col-12 g-col-md-4">
+      <:card-header>
+        <p>Here is a title</p>
+      </:card-header>
+      <:card-body>
+        <p>Here is some body content</p>
+      </:card-body>
+    </Container.Card>
+    <Container.Card @onClick={{this.clickHandler}} class="g-col-12 g-col-md-4">
+      <:card-header>
+        <p>Here is a title</p>
+      </:card-header>
+      <:card-body>
+        <p>Here is some body content</p>
+      </:card-body>
+    </Container.Card>
   </Mktg::CardContainer>
+  <div class="container">
+    <p class="pt-3 fw-bold">Vertical card with right aligned price</p>
+    <Mktg::CardContainer class="justify-content-center" as |Container|>
+      <Container.VerticalCard
+        class="g-col-12"
+        @title="The Gig"
+        @subtitle="Experience lightning-fast internet connectivity"
+        @price="300"
+      >
+        <p class="card-text mt-2">Includes:</p>
+        <div class="container">
+          <div class="row row-cols-2">
+            <p class="col-12 col-md-6"><span
+                class="me-2 fw-bold"
+              >&#x2713;</span>Cool feature</p>
+            <p class="col-12 col-md-6"><span
+                class="me-2 fw-bold"
+              >&#x2713;</span>Cool feature</p>
+            <p class="col-12 col-md-6"><span
+                class="me-2 fw-bold"
+              >&#x2713;</span>Cool feature</p>
+            <p class="col-12 col-md-6"><span
+                class="me-2 fw-bold"
+              >&#x2713;</span>Cool feature</p>
+          </div>
+        </div>
```

Comment:
I'll make a note in the feature list card I'm working on now. I believe we can come back and use that component here once it's ready

Reply:
Yes that was my plan once it was finished!